app/cli.py

```diff
@@ -115,7 +115,7 @@ def create_resource(resource, db):
         print('exception', e)
 
 
-def update_resource(resource, existing_resource):
+def update_resource(resource, existing_resource):   # pragma: no cover
```

Comment:
Did you want to do this to all the CLI functions or did you find non-cli usages of them?

Reply:
I only checked the ones that weren't already covered. 
I'll go back over it because there is one more I could probably make exempt.